156 move config files used for testing to test folder and create two config files for showcasing

QCHR1581 opened the pull request:
- Moved config files which are used for testing purposes to a folder named **config_files** within the **test** directory.
- Created the config file **trihard.conf** which is used as default config file for webserv
- Renamed **standard_config.conf** to **example.conf** which gets used in integration tests and for showcasing the capability of webserv

Close #156

gwolf-011235 reviewed (changes requested):
I would delete target "run" as we can just ./webserv now
Then we only have one CONFIGFILE variables aka delete CONFIGFILE_INTEGRATION
You need to update example.conf with values from integration_test_config.conf (I changed it and added some stuff)

QCHR1581 commented:
> I would delete target "run" as we can just ./webserv now Then we only have one CONFIGFILE variables aka delete CONFIGFILE_INTEGRATION You need to update example.conf with values from integration_test_config.conf (I changed it and added some stuff)

The target run from the makefile is deleted as well as CONFIGFILE_INTEGRATION (CONFIGFILE gets used instead). 

I already updated the example.conf file with the values from the integration_test_config.conf

gwolf-011235 reviewed (approved):
LGTM
